### PR TITLE
OSDOCS-11300: hardcoded 4.17 version number bump

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.17+

Issue:
https://issues.redhat.com/browse/OSDOCS-11300

Link to docs preview:
In progress

QE review:
This PR bumps hard coded version numbers; no SME/QE review required.

Additional information:
Touches a lot of distributions and associated projects.

### CODE REVIEW

**Review by kowen-rh — COMMENTED** [review]
I'm a little confused about some of the versions used in the MicroShift sections, but they might be fine. Other than that, the changes look good to me! :+1: 

/remove-label peer-review-in-progress
/label peer-review-done

**Comment by sheriff-rh** [comment]
Good points, @kowen-rh. Thank you!

@ShaunaDiaz would you mind reviewing as well since this touches a lot of microshift content? No rush. Thanks!

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 1]
```diff
@@ -58,14 +58,14 @@ The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you
 [source,text]
 ----
 name = "microshift_blueprint"
-description = "MicroShift 4.16.1 on x86_64 platform"
+description = "MicroShift 4.17.1 on x86_64 platform"
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/microshift-adding-service-to-blueprint.adoc:61]: MS is skipping 4.17.0.

File: modules/microshift-get-rpm-release-info.adoc [hunk 2]
```diff
@@ -19,25 +19,25 @@ To see the images included with a {microshift-short} release, you can list the c
...
-microshift-release-info-0:4.16.10-202408291007.p0.g6e4ee4d.assembly.4.16.10.el9.noarch
-microshift-release-info-0:4.16.2-202407040825.p0.g2e0407e.assembly.4.16.2.el9.noarch
-microshift-release-info-0:4.16.3-202407111123.p0.ge4206d3.assembly.4.16.3.el9.noarch
-microshift-release-info-0:4.16.4-202407191908.p0.g057a9af.assembly.4.16.4.el9.noarch
-microshift-release-info-0:4.16.5-202407250951.p0.g0afcb57.assembly.4.16.5.el9.noarch
-microshift-release-info-0:4.16.6-202408010822.p0.gc4ded66.assembly.4.16.6.el9.noarch
-microshift-release-info-0:4.16.7-202408081107.p0.g0597bb8.assembly.4.16.7.el9.noarch
-microshift-release-info-0:4.16.8-202408150851.p0.gc8a3bb1.assembly.4.16.8.el9.noarch
-microshift-release-info-0:4.16.9-202408220842.p0.gefa92a2.assembly.4.16.9.el9.noarch
+microshift-release-info-0:4.17.0-202406260523.p0.gc5a37df.assembly.4.17.0.el9.noarch
```
Comment (ShaunaDiaz) [inline modules/microshift-get-rpm-release-info.adoc:30]:
```suggestion
```
Easier to delete this output line than to revert all of the others.

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 3]
```diff
@@ -48,7 +48,7 @@ EOF
 +
 [NOTE]
 ====
-The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.16.0` to download the {microshift-short} 4.16.0 RPMs.
+The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.17.0` to download the {microshift-short} 4.17.0 RPMs.
```
Comment (ShaunaDiaz) [inline modules/microshift-adding-service-to-blueprint.adoc:51]:
```suggestion
The wildcard `*` in the commands uses the latest {microshift-short} RPMs. If you need a specific version, substitute the wildcard for the version you want. For example, insert `4.17.1` to download the {microshift-short} 4.17.1 RPMs.
```
There is no 4.17.0 release for MicroShift. It is coming out 4.17.1. We have to rebase on the released version of LVMS, so MicroShift is planned as a .1 z-stream release. cc @kowen-rh

File: modules/microshift-adding-service-to-blueprint.adoc [hunk 4]
```diff
@@ -115,11 +115,11 @@ $ sudo composer-cli blueprints depsolve __<microshift_blueprint>__ | grep micros
 [source,terminal]
 ----
 blueprint: microshift_blueprint v0.0.1
-    microshift-greenboot-4.16.1-202305250827.p0.g4105d3b.assembly.4.16.1.el9.noarch
-    microshift-networking-4.16.1-202305250827.p0.g4105d3b.assembly.4.16.1.el9.x86_64
-    microshift-release-info-4.16.1-202305250827.p0.g4105d3b.assembly.4.16.1.el9.noarch
-    microshift-4.16.1-202305250827.p0.g4105d3b.assembly.4.16.1.el9.x86_64
-    microshift-selinux-4.16.1-202305250827.p0.g4105d3b.assembly.4.16.1.el9.noarch
+    microshift-greenboot-4.17.1-202305250827.p0.g4105d3b.assembly.4.17.1.el9.noarch
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-adding-service-to-blueprint.adoc:118]: 4.17.1 is correct

File: modules/microshift-etcd-version.adoc [hunk 5]
```diff
@@ -21,7 +21,7 @@ $ microshift-etcd version
 .Example output
 [source,terminal,subs="attributes+"]
 ----
-microshift-etcd Version: 4.16.0
+microshift-etcd Version: 4.17.0
```
Comment (ShaunaDiaz) [inline modules/microshift-etcd-version.adoc:24]:
```suggestion
microshift-etcd Version: 4.17.1
```

File: modules/microshift-get-rpm-release-info.adoc [hunk 6]
```diff
@@ -48,15 +48,15 @@ microshift-release-info-0:4.16.9-202408220842.p0.gefa92a2.assembly.4.16.9.el9.no
 ----
 $ sudo dnf download microshift-release-info-_<release_version>_ # <1>
 ----
-<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, for example, `4.16.0`.
+<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, for example, `4.17.0`.
```
Comment (ShaunaDiaz) [inline modules/microshift-get-rpm-release-info.adoc:51]:
```suggestion
<1> Replace `_<release_version>_` with the numerical value of the release you are deploying, using the entire version number, for example, `4.17.1`.
```

File: modules/microshift-get-rpm-release-info.adoc [hunk 7]
```diff
@@ -48,15 +48,15 @@ microshift-release-info-0:4.16.9-202408220842.p0.gefa92a2.assembly.4.16.9.el9.no
...
 --
 +
 --
 .Example output
 [subs="+quotes"]
 [source,terminal]
 ----
-microshift-release-info-4.16.0.-202311101230.p0.g7dc6a00.assembly.4.16.0.el9.noarch.rpm # <1>
+microshift-release-info-4.17.0.-202311101230.p0.g7dc6a00.assembly.4.17.0.el9.noarch.rpm # <1>
```
Comment (ShaunaDiaz) [inline modules/microshift-get-rpm-release-info.adoc:59]:
```suggestion
microshift-release-info-4.17.1.-202311101230.p0.g7dc6a00.assembly.4.17.1.el9.noarch.rpm # <1>
```

File: modules/microshift-oc-mirror-embed-ops-disconnected-use.adoc [hunk 8]
```diff
@@ -120,14 +120,14 @@ source = "registry.redhat.io/redhat/redhat-operator-index@sha256:7a76c0880a83903
...
-description = "MicroShift 4.16.1 on x86_64 platform"
+description = "MicroShift 4.17.1 on x86_64 platform"
 version = "0.0.1"
 modules = []
 groups = []
 
 [[packages]] <1>
 name = "microshift"
-version = "4.16.1"
+version = "4.17.1"
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-oc-mirror-embed-ops-disconnected-use.adoc:130]: 4.17.1

File: modules/microshift-updates-troubleshooting.adoc [hunk 9]
```diff
@@ -32,9 +32,9 @@ Check the following update paths:
 
 *{product-title} update paths*
 
-* Generally Available Version 4.16.0 to 4.16.z on {op-system-ostree} 9.4
-* Generally Available Version 4.15.0 from {op-system-base} 9.2 to 4.16.0 on {op-system-base} 9.4
-* Generally Available Version 4.14.0 from {op-system-base} 9.2 to 4.16.0 on {op-system-base} 9.4
+* Generally Available Version 4.17.0 to 4.17.z on {op-system-ostree} 9.4
```
Comment (ShaunaDiaz) [inline modules/microshift-updates-troubleshooting.adoc:35]:
```suggestion
* Generally Available Version 4.17.1 to 4.17.z on {op-system-ostree} 9.4
```

File: modules/microshift-updates-troubleshooting.adoc [hunk 10]
```diff
@@ -32,9 +32,9 @@ Check the following update paths:
...
+* Generally Available Version 4.15.0 from {op-system-base} 9.2 to 4.17.0 on {op-system-base} 9.4
```
Comment (ShaunaDiaz) [inline modules/microshift-updates-troubleshooting.adoc:36]:
```suggestion
* Generally Available Version 4.15.0 from {op-system-base} 9.2 to 4.16.0 on {op-system-base} 9.4
```
do not change

File: modules/microshift-updates-troubleshooting.adoc [hunk 11]
```diff
@@ -32,9 +32,9 @@ Check the following update paths:
...
+* Generally Available Version 4.14.0 from {op-system-base} 9.2 to 4.17.0 on {op-system-base} 9.4
```
Comment (ShaunaDiaz) [inline modules/microshift-updates-troubleshooting.adoc:37]:
```suggestion
* Generally Available Version 4.14.0 from {op-system-base} 9.2 to 4.15.0 on {op-system-base} 9.4
```
do not change

File: modules/microshift-updates-troubleshooting.adoc (hunk 9 above)
Comment (ShaunaDiaz) [reply modules/microshift-updates-troubleshooting.adoc:35]: Trickier area because we have some skip-version releases, but others are not.

**Review by ShaunaDiaz — COMMENTED** [review]
MicroShift is not releasing at 4.17.0, we are going out 4.17.1 because of a needed rebase. Some version changes are cool, others aren't. Should have got them all, but lmk if you think I missed one. TY!!

**Comment by sheriff-rh** [comment]
All changes to microshift complete; merging to 4.17+.

Thank you all for your reviews!

**Pull request merged**